server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementIterator.java

```diff
@@ -214,8 +214,12 @@ private static Map<TableId,MergeInfo> parseMerges(final String merges) {
 
   private static boolean shouldReturnDueToSplit(final TabletMetadata tm,
       final long splitThreshold) {
-    return tm.getFilesMap().values().stream().map(DataFileValue::getSize)
-        .collect(Collectors.summarizingLong(Long::longValue)).getSum() > splitThreshold;
+    final long sumOfFileSizes =
```

Comment:
When there is an operation id set we can ignore the tablet for automatic split. 

```suggestion
    if (tm.getOperationId() != null) {
      // tablet may be currently splitting or merging, can ignore it until that finishes
      return false;
    }

    final long sumOfFileSizes =
```

Reply:
In addition to this check early on in the tserver, I added a check that is as late as possible in the manager.  I added that in c8d39de73d7984a490c3693aa9760c96546b1bb2 w/o doing a PR.  Its such a small change and this PR does not modify that file.  Let me know if you see any problems w/ that little change.

Reply:
You mention for `split`, but do you think that TabletManagementIterator should not return the TabletMetadata for any condition when there is an operationId set?

Reply:
There should not be a need to assign a tablet with an op id.  So for the location update case it does seem like there is no need to return it.  Merge is going to change a lot, so not sure about it at the moment.

Also a tablet with an op id should never have a location.  Maybe we could add another check for the bad state case.

Reply:
> So for the location update case it does seem like there is no need to return it. Merge is going to change a lot, so not sure about it at the moment.

It might need to be returned so that it can be unassigned?

Reply:
I added this check in cfced0b, but the iterator still returns location updates when opid != null, for now.

Reply:
Created #3443